datalake/src/test/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreServiceTest.java

```diff
@@ -444,6 +444,76 @@ public void testIsDatalakeInRestoreWithOtherRuntimeExceptions() {
         assertFalse(sdxBackupRestoreService.isDatalakeInRestoreProgress(sdxCluster.getClusterName(), USER_CRN));
     }
 
+    @Test
+    public void testModifyBackupLocationWithAWS() {
+        DetailedEnvironmentResponse detailedEnvironmentResponse = new DetailedEnvironmentResponse();
+        detailedEnvironmentResponse.setCloudPlatform("AWS");
+        when(environmentClientService.getByName(anyString())).thenReturn(detailedEnvironmentResponse);
+        SdxCluster sdxCluster = getValidSdxCluster();
+        sdxCluster.setRangerRazEnabled(true);
+        assertEquals("s3:/bucket", sdxBackupRestoreService.modifyBackupLocation(sdxCluster, "s3:/bucket"));
```

Comment:
this is not a valid s3 path, isn't it?

Reply:
@votez Thanks a lot for checking!! This test including the other two you comment out is just trying to make sure if the passed in location is not valid, we will just returning the original passed in location. Please let me know if this makes sense. Thank you!

Reply:
@votez I still added in a few invalid locations but they are will just be caught by `URISyntaxException`. Let me know if they look good. Thank you!